src/rpc/server/scala/api/src/main/scala/com/ctc/omega_edit/api/Change.scala

```diff
@@ -35,7 +35,5 @@ object Change {
   case object Undefined extends Op
 
   sealed trait Result
-  case object Paused extends Result
-  case object Fail extends Result
```

Comment:
Why are `Paused` and `Fail` being removed?

Reply:
Because they need to be handled in the _client_, not in the server.